### DISCUSSION TITLE
IPopupHost ConfigurePosition marked obsolete

### DISCUSSION THREAD

**Discussion by aguahombre** [AvaloniaUI/Avalonia#17579] [Q&A]
### Describe the bug

The ```void ConfigurePosition(PopupPositionRequest positionRequest);``` method is marked as obsolete.
This API may be removed in Avalonia 12. If you depend on this API, please open an issue with details of your use-case.

I am using this method to position a custom on-screen touch keypad overlay so that it does not obscure the TextBox displaying the data entry.

This was working in v10 and before and still works in v11 but now marked obsolete.
Is there a replacement way to do this in Avalonia 11/12?

I'm not sure if Bug is the correct label.

### To Reproduce

N/a

### Avalonia version

11.2.1

### OS

Windows, Linux

**Comment by maxkatz6** [comment]
>Is there a replacement way to do this in Avalonia 11/12?

If other placement properties are not enough, you can use `Popup.CustomPopupPlacementCallback`
